builddefs/converters.mk

```
@@ -0,0 +1,26 @@
+ifeq ($(strip $(CTPC)), yes)
+    CONVERT_TO_PROTON_C=yes
+endif
+ifeq ($(strip $(CONVERT_TO_PROTON_C)), yes)
+    CONVERT_TO=proton_c
+endif
+
+# TODO: opt in rather than assume everything uses a pro micro
+PIN_COMPATIBLE ?= promicro
+ifneq ($(CONVERT_TO),)
+    # glob to search each platfrorm and/or check for valid converter
+    CONVERTER := $(wildcard $(PLATFORM_PATH)/*/converters/$(PIN_COMPATIBLE)_to_$(CONVERT_TO)/)
+    ifeq ($(CONVERTER),)
+        $(call CATASTROPHIC_ERROR,Converting to '$(CONVERT_TO)' not possible!)
```

```suggestion
        $(call CATASTROPHIC_ERROR,Converting from '$(PIN_COMPATIBLE)' to '$(CONVERT_TO)' not possible!)
```

Just a nit, outputting the assumed board/pinout could aid in troubleshooting later.

---

Good point!